packages/playwright/src/plugins/gitCommitInfoPlugin.ts

```diff
@@ -41,12 +41,19 @@ function debug(s: string, ...args: any[]) {
 }
 
 const gitCommitInfoPlugin = (fullConfig: FullConfigInternal): TestRunnerPlugin => {
+  let storedConfig: FullConfig;
+  let storedConfigDir: string;
+
   return {
     name: 'playwright:git-commit-info',
 
     setup: async (config: FullConfig, configDir: string) => {
       const metadata = config.metadata as MetadataWithCommitInfo;
+      storedConfig = config;
+      storedConfigDir = configDir;
```

Comment:
looking at it again, there's no need to store configDir separately - you can use `config.configDir` instead